### PR TITLE
Create Panda's Bio

### PR DESCRIPTION
**Pull request by IPandaType**
@githubteacher, This pull request adds a bio for @codingpanda7 as requested in #10 

Please review and merge

### CODE REVIEW

**Comment by IPandaType** [comment]
A comment

File: docs/codingpanda7bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Panda's Bio
+
+- **Name:** Pandu
+- **Occupation:** Police Officer
+- **Location:** San Diego
```
Comment (IPandaType) [inline docs/codingpanda7bio.md:5]: Nice Place

Comment (richng) [inline docs/codingpanda7bio.md:5]: Agreed.  Love San Diego.

**Comment by IPandaType** [comment]
:shipit:

**Pull request merged**